core/src/net/sf/openrocket/rocketcomponent/FinSet.java

```diff
@@ -707,9 +707,14 @@ public double getRotationalUnitInertia() {
 			h = MathUtil.safeSqrt(h * singlePlanformArea/ w);
 		}
 
+		final double inertia = h * h / 12;
```

Comment:
nit:  in an entire function about calculating inertia,  naming a variable just "inertia" is a bit vague.

Recommend better naming or at least a comment about which inertia this is.
(afaict, it's an approximation for a "thin rod about it's center": 
https://d2vlcm61l7u1fs.cloudfront.net/media%2Fe27%2Fe2782b33-c9c3-4e4a-aac3-1e245d5167a1%2FphpRvs8d4.png )

Reply:
But, to be clear, pulling this out into a separate variable is probably a good thing, and adds to readability  (in addition to correct code ;)